README.md

```diff
@@ -550,6 +550,7 @@ See also [Are we (I)DE yet?](https://areweideyet.com/) and [Rust Tools](https://
 
   * [Atom](https://atom.io/)
     * [zargony/atom-language-rust](https://github.com/zargony/atom-language-rust)
+    * [rust-lang/atom-ide-rust](https://github.com/rust-lang/atom-ide-rust)
```

Comment:
Thanks for the submission. Can you please add a description for the repository (I know the other one above it doesn't have one, but we're requiring this for newer submissions now) and add the Travis build badge for it as I'm seeing a build at https://travis-ci.com/github/rust-lang/atom-ide-rust/builds.

Reply:
Hi. I have made the changes. Pls take a look